wallets/src/main/java/bisq/wallets/stores/BitcoinWalletStore.java

```diff
@@ -21,34 +21,46 @@
 import bisq.common.proto.ProtoResolver;
 import bisq.common.proto.UnresolvableProtobufMessageException;
 import bisq.persistence.PersistableStore;
+import bisq.wallets.RpcConfig;
 import com.google.protobuf.InvalidProtocolBufferException;
 import lombok.Getter;
+import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 
 import java.util.HashSet;
+import java.util.Optional;
 import java.util.Set;
 
 @Slf4j
 public class BitcoinWalletStore implements PersistableStore<BitcoinWalletStore> {
+    @Getter
+    @Setter
+    private Optional<RpcConfig> rpcConfig = Optional.empty();
```

Comment:
Ah the RpcConfig is using inside the walletstores. In that case ignore my previous comment, but I think RpcConfig does not require to implement PersistableStore if it is not stored itself. PersistableStore is used for the classes which used for the storage file.

Reply:
You're right the `RpcConfig` does not need to implement `PersistableStore`. I updated the PR.